atd-vze/src/queries/gqlAbstract.js

```diff
@@ -122,6 +133,15 @@ gqlAbstractTableAggregateName (
     delete this.config["where"][key];
   }
 
+  /**
+   * Removes a column from the or condition
+   * @param {string} key - The name of the column
```

Comment:
maybe i'm confused but I think the documentation/comments here should say that the param is called `orObject`

Reply:
and I'm thinking its probably and `{object}` not a string?

Reply:
Good catch! That is my copypasta. Fixed this for the future.